internal/usecase/daemon/coderabbit_regression_test.go

```
@@ -303,7 +304,154 @@ func TestResumeStoppedSessionRejectsReplacementDuringCatalogueRead(t *testing.T)
+	replacement.IncarnationID = domain.IncarnationID{9}
+	replacement.CreatedAt++
+	replacement.UpdatedAt = replacement.CreatedAt
+	require.NoError(t, catalogue.Replace(replacement.Name, replacement))
+	d.mu.Lock()
+	d.inactive[replacement.Name] = inactiveSessionFromRecord(replacement, ports.SessionDown, nil)
+	d.mu.Unlock()
+	close(catalogue.release)
+
+	var protocol *protoErr
+	require.ErrorAs(t, <-result, &protocol)
+	require.Equal(t, uint16(ports.ErrNoSuchTarget), protocol.code)
```

golangci-lint (2.12.2)

[error] unnecessary conversion

(unconvert)